internal/olm/operator/registry/operator_installer.go

```diff
@@ -191,15 +198,17 @@ func (o OperatorInstaller) getOperatorGroup(ctx context.Context) (*v1.OperatorGr
 	return &ogList.Items[0], true, nil
 }
 
-func (o OperatorInstaller) createSubscription(ctx context.Context, cs *v1alpha1.CatalogSource) error {
+func (o OperatorInstaller) createSubscription(ctx context.Context, cs *v1alpha1.CatalogSource) (*v1alpha1.Subscription, error) {
 	sub := newSubscription(o.StartingCSV, o.cfg.Namespace,
 		withPackageChannel(o.PackageName, o.Channel, o.StartingCSV),
-		withCatalogSource(cs.GetName(), o.cfg.Namespace))
+		withCatalogSource(cs.GetName(), o.cfg.Namespace),
+		withInstallPlanApproval(v1alpha1.ApprovalManual))
```

Comment:
I'm not sure if CI will pass because installplans created by `run packagemanifests` will not be approved. IIRC we discussed that automatic approval is a bug in `run packagemanifests` since a version is specified but that version might not be the channel head specified in a `Subscription`, and setting approval to manual is correct for both the index and configmap `CatalogCreator`'s; we would need to approve _all_ `Subscription`s manually with this change.

Reply:
After the catalog source is created, the logic should be identical between `run packagemanifests` and `run bundle`.

We shouldn't have different implementations of the `InstallPlanApprover` interface (and we probably don't need that interface at all).